rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-instance-types
+[id="rosa-hcp-instance-types"]
+= ROSA with HCP instance types
+
+All {hcp-title} clusters require a minimum of 2 worker nodes. All {hcp-title} clusters support a maximum of 90 worker nodes. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss.
+
+[NOTE]
+====
+Approximately one vCPU core and 1 GiB of memory are reserved on each worker node and removed from allocatable resources. This reservation of resources is necessary to run processes required by the underlying platform. These processes include system daemons such as udev, kubelet, and container runtime among others. The reserved resources also account for kernel reservations.
+
+{OCP} core systems such as audit log aggregation, metrics collection, DNS, image registry, SDN, and others might consume additional allocatable resources to maintain the stability and maintainability of the cluster. The additional resources consumed might vary based on usage.
+
+For additional information, see the link:https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#system-reserved[Kubernetes documentation].
+====
+
+include::modules/rosa-sdpolicy-am-aws-compute-types.adoc[leveloffset=+1]
+include::modules/rosa-sdpolicy-am-aws-compute-types-graviton.adoc[leveloffset=+1]
```

Comment:
Small FYI, and likely irrelevant with us moving away from Asciidoc, but I had read that you want to put spaces between `include` modules when I was having issues last week. It did solve my issues, and no spacing can basically cause them to bleed together.

No need to change anything; just a heads up.

Reply:
I changed it just in case